data/endpoints/fps-initiate-payment-v3.json

```diff
@@ -1,777 +1,816 @@
+          "BatchPaymentInstructionCounterpartAccount": {
+              "required": [
+                  "identification"
+              ],
+              "type": "object",
+              "properties": {
+                  "identification": {
+                      "$ref": "#/components/schemas/BatchPaymentInstructionCounterpartAccountIdentification",
+                      "description": "The identifiable information of an account."
+                  }
+              },
+              "additionalProperties": false,
+              "description": "Information about the counterpart in a given transaction."
+          },
+          "BatchPaymentInstructionCounterpartAccountIdentification": {
+              "type": "object",
+              "properties": {
+                  "iban": {
+                      "type": "string",
+                      "description": "The International Bank Account Number of the account.",
+                      "nullable": true
+                  },
+                  "other": {
+                      "$ref": "#/components/schemas/BatchCounterpartAccountGenericIdentification",
+                      "description": "Unique identification of an account, as assigned by the account servicer, using an identification scheme.",
+                      "nullable": true
+                  }
+              },
+              "additionalProperties": false,
+              "description": "The identifiable information of an account."
+          },
+          "BatchRemittanceInformation": {
+              "type": "object",
+              "properties": {
+                  "structured": {
+                      "$ref": "#/components/schemas/BatchStructured",
+                      "description": "Information supplied by the remitter (in a structured form), to reconcile an entry with item(s) that the payment intends to settle (e.g., a purchase reference number).",
+                      "nullable": true
+                  }
+              },
+              "additionalProperties": false,
+              "description": "Information supplied to enable the matching/reconciliation of an entry with the items that the payment is intended to settle, such as commercial invoices in an accounts’ receivable system."
+          },
+          "BatchStructured": {
+              "type": "object",
+              "properties": {
+                  "creditorReferenceInformation": {
+                      "$ref": "#/components/schemas/BatchCreditorReferenceInformation",
+                      "description": "Reference information provided by the ultimate debtor to allow the identification of underlying documents by the creditor.",
+                      "nullable": true
+                  }
+              },
+              "additionalProperties": false,
+              "description": "Information supplied by the remitter (in a structured form), to reconcile an entry with item(s) that the payment intends to settle (e.g., a purchase reference number)."

```

Comment:
Do we use e.g.? I thought it was against MS Style Guide. I'd say, (_for example_, a purchase reference number)

Reply:
```suggestion
              "description": "Information supplied by the remitter (in a structured form), to reconcile an entry with item(s) that the payment intends to settle (for example, a purchase reference number)."

```